README.md

```diff
@@ -53,53 +53,54 @@ import KlaviyoSwift
 2. To add Klaviyo's tracking functionality, include the following line in AppDelegate.swift, within `application:didFinishLaunchingWithOptions`:
 
 ```swift
-Klaviyo.setupWithPublicAPIKey(apiKey: "YOUR_KLAVIYO_PUBLIC_API_KEY")
+KlaviyoSDK().initalize(with: "YOUR_KLAVIYO_PUBLIC_API_KEY")
 ```
 
-3. Begin tracking events anywhere within your application by calling the `trackEvent` method in the relevant location.
+3. Begin tracking events anywhere within your application by calling the `create(event:)` method in the relevant location.
 
 ```swift
-let klaviyo = Klaviyo.sharedInstance
-
-let customerDictionary : NSMutableDictionary = NSMutableDictionary()
-customerDictionary[klaviyo.KLPersonEmailDictKey] = "john.smith@example.com"
-customerDictionary[klaviyo.KLPersonFirstNameDictKey] = "John"
-customerDictionary[klaviyo.KLPersonLastNameDictKey] = "Smith"
-
-let propertiesDictionary : NSMutableDictionary = NSMutableDictionary()
-propertiesDictionary["Total Price"] = 10.99
-propertiesDictionary["Items Purchased"] = ["Milk","Cheese", "Yogurt"]
-Klaviyo.sharedInstance.trackEvent(
-    eventName: "Completed Checkout",
-    customerProperties: customerDictionary,
-    properties: propertiesDictionary
-)
+let klaviyo = KlaviyoSDK()
```

Comment:
Oh so it is OK to create an instance of KlaviyoSDK repeatedly? I.e. they don't have to just create one and reuse that instance?

Reply:
(not a critique, just catching up, I haven't paid as close attention as I could've to the 2.0 update)

Reply:
Yup I've moved away from the singleton pattern since it's not needed. That's all handled internally.

Reply:
nice

Reply:
It's also a struct now vs a class.